en/flight_controller/pixhawk5x.md

```diff
@@ -172,6 +172,10 @@ Under these conditions the system will not draw any power (will not be operation
 1. **USB** input (operational range 4.1V to 5.7V, 0V to 6V undamaged)
 1. Servo input: VDD_SERVO pin of **FMU PWM OUT** and **I/O PWM OUT** (0V to 42V undamaged)
 
+:::note
+If you need analog battery voltage monitoring, you can connect an ADS1115 ADC via I2C, enable it via [ADC_ADS1115_EN](../advanced_config/parameter_reference.md#ADC_ADS1115_EN) and pull down the nPOWER_IN_A gpio input.
+:::
+
```

Comment:
Proposal, we mention I2C monitoring here too.
```suggestion
:::note
I2C battery monitoring is enabled by default.
If analog battery voltage monitoring is needed, you can connect an ADS1115 ADC via I2C, enable it via [ADC_ADS1115_EN](../advanced_config/parameter_reference.md#ADC_ADS1115_EN), and pull down the `nPOWER_IN_A` GPIO input.
:::

```
Questions:
-  "pull down the `nPOWER_IN_A` GPIO input" means attach that input to ground right?
- where is the nPOWER_IN_A input? I can't see it on the above diagrams.

Reply:
I would say `Digital I2C battery monitoring`

> "pull down the nPOWER_IN_A GPIO input" means attach that input to ground right?

Yes

> where is the nPOWER_IN_A input? I can't see it on the above diagrams.

Not sure for this board. @AntonioAguilera?

> Is there a good "canonical link for ADS1115? I couldn't see one, but if we can find a good description it might be good to link

It is common, but I don't know if there's a canonical one.

Reply:
@AntonioAguilera Can you tell me where the `nPOWER_IN_A` input is on the Holybro Pixhawk 5X?

@bkueng If Antonio can't answer, is there someone else we can ask? Hint is not much use without this information.

Reply:
```suggestion
**Voltage monitoring**

Digital I2C battery monitoring is enabled by default (see [Quickstart > Power](../assembly/quick_start_pixhawk5x.md#power)).

:::note
Analog battery monitoring via an ADC is not supported on this particular board, but may be supported in variations of this flight controller with a different baseboard.
:::

```